vite.config.ts

```diff
@@ -3,13 +3,21 @@ import { cloudflare } from "@cloudflare/vite-plugin";
 import tailwindcss from "@tailwindcss/vite";
 import { defineConfig } from "vite";
 import tsconfigPaths from "vite-tsconfig-paths";
+import { nodePolyfills } from "vite-plugin-node-polyfills";
 
 export default defineConfig({
 	plugins: [
-		cloudflare({ viteEnvironment: { name: "ssr" } }),
 		tailwindcss(),
 		reactRouter(),
 		tsconfigPaths(),
+		nodePolyfills({
+			globals: {
+				Buffer: true,
+				global: true,
+				process: true,
```

Comment:
process should not be needed. Global probably as well. AFAIR, only buffer is needed.

Reply:
Removed